internal/monitors/appmesh/metadata.yaml

```diff
@@ -0,0 +1,272 @@
+    type: cumulative
+  - description: Total active requests
+    included: true
+    name: gauge.<ingress/egress>.upstream_rq_active
+    type: gauge
+  - description: Total requests pending a connection pool connection
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_pending_total
+    type: cumulative
+  - description: Total requests that overflowed connection pool circuit breaking and were failed
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_pending_overflow
+    type: cumulative
+  - description: Total requests that were failed due to a connection pool connection failure
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_pending_failure_eject
+    type: cumulative
+  - description: Total active requests pending a connection pool connection
+    included: true
+    name: gauge.<ingress/egress>.upstream_rq_pending_active
+    type: gauge
+  - description: Total requests cancelled before obtaining a connection pool connection
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_cancelled
+    type: cumulative
+  - description: Total requests that resulted in an immediate 503 due to maintenance mode
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_maintenance_mode
+    type: cumulative
+  - description: Total requests that timed out waiting for a response
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_timeout
+    type: cumulative
+  - description: Total requests that hit the per try timeout
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_per_try_timeout
+    type: cumulative
+  - description: Total requests that were reset remotely
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_rx_reset
+    type: cumulative
+  - description: Total requests that were reset locally
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_tx_reset
+    type: cumulative
+  - description: Total request retries
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_retry
+    type: cumulative
+  - description: Total request retry successes
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_retry_success
+    type: cumulative
+  - description: Total requests not retried due to circuit breaking
+    included: true
+    name: derive.<ingress/egress>.upstream_rq_retry_overflow
+    type: cumulative
+  - description: Total external origin requests completed
+    included: true
+    name: derive.<ingress/egress>.external.upstream_rq_completed
+    type: cumulative
+  - description: External origin aggregate HTTP response codes
+    included: true
+    name: derive.<ingress/egress>.external.upstream_rq_<_xx>
+    type: cumulative
+  - description: External origin specific HTTP response codes
+    included: true
+    name: derive.<ingress/egress>.external.upstream_rq_<_>
+    type: cumulative
+  - description: External origin request time milliseconds
+    included: true
```

Comment:
We are going to use all these metrics in the built-in content?

Reply:
Good point. I'll turn off what's not being used for the built-in content